src/scripts/app.js

```diff
@@ -795,6 +815,31 @@ export default class InteractiveBook extends H5P.EventDispatcher {
       }
     });
 
+    this.autoAdvance = (direction, toTop) => {
```

Comment:
This function is kind of confusing to read, something like this might be easier:

```
function go(direction, scrollToTop) {
  const nextSlide = Math.max(0, Math.min(this.chapters.length, this.activeChapter + (direction === 'next' ? 1 : -1)));

  if (this.hasSummary() && nextSlide === this.chapters.length) {
    // go to summary
    return;
  }

  // do other stuff
}
```